Makefile

```diff
@@ -141,6 +141,11 @@ test-acceptance-core-webui: ## Run core webUI acceptance tests
 test-acceptance-core-webui: $(acceptance_test_deps)
 	BEHAT_BIN=$(BEHAT_BIN) ../../tests/acceptance/run.sh --remote --type webui -c ../../tests/acceptance/config/behat.yml --tags '~@skipOnEncryption&&~@skipOnEncryptionType:${ENCRYPTION_TYPE}&&~@skip&&~@app-required'
 
+.PHONY:
+starlark-generate:
+starlark-generate:
```

Comment:
Is this intended that `starlark-generate` is present twice?

Reply:
it's a mistake I'll remove this